File: code/0121 Best Time to Buy and Sell Stock.cpp
int maxProfit(int* prices, int pricesSize) {
    int max=0;
    for(int i=0;i<pricesSize;i++){
        for(int j=i+1;j<pricesSize;j++){
            if(prices[j]-prices[i]>max){
                max=prices[j]-prices[i];
            }
        }
    }
    return max;
}

File: code/0121 Best Time to Buy and Sell Stock.py
class Solution(object):
    def maxProfit(self, prices):
        maxi = buy = sell = 0
        while sell < len(prices):
            if prices[buy] <= prices[sell]:
                maxi = max(maxi, prices[sell] - prices[buy])
                sell += 1
            else:
                buy = sell
        return maxi
